### ISSUE TITLE
Not working with Core-Module v2.0.4

### ISSUE THREAD

**Issue by necon0951** [pwatson100/token-action-hud-alien#4]
Hello, the module stops working as soon as the new Action-Hud-Core version 2.0.4 is out. Is it possible to fix it?

**Comment by pwatson100** [comment]
That is correct. The latest version of TAH Core has a number of breaking changes and is not compatable with older versions of our modules. You will need to downgrade to the last version of TAH Core untill I have a chance to asses and make the required changes.

**Comment by Daddldiddl** [comment]
I just had this issue setting up a new Alien RPG server for my group. Core Module 2.x (at V2.0.10 already) has been out for a while now - any time frame on when this module will be made compatible?

**Comment by pwatson100** [comment]
I have an updated version I am testing.  I hope to release it out in the next two weeks.

If you want you can download the updated version from the TAH-V2-Version branch.

**Comment by LuvacZantor** [comment]
Is the updated version ready? 
![image](https://github.com/user-attachments/assets/c30bf878-1f1a-43da-9789-9d4704595a99)

**Comment by LuvacZantor** [comment]
is there a manifest link for TAH-V2-Version branch? I'm moving to a hosting provider and do not have direct access to the Data/Modules folder

**Comment by LuvacZantor** [comment]
Has there been any progress on releasing a compatible Module?

**Comment by pwatson100** [comment]
The new version for TAH V2 compatibility has been released.
